scripts/assets/file_content_validation_checks.textproto

```diff
@@ -195,8 +195,9 @@ file_content_checks {
 file_content_checks {
   file_path_regex: ".+?\\.kt"
   prohibited_content_regex: "\\sAppCompatActivity\\("
```

Comment:
This & the message should probably be updated. We actually want to restrict with AppCompatActivity and the new base InjectableAppCompatActivity (per my feedback above re: having a common base activity class). The error message should indicate that either the auot-localized one OR the system one should be used.

We probably should also add a check (& test for it) to force an allow-list for classes that inherit from the system language (similar to what we do for using the parameterized test runner) since such cases should always be special case and require additional reviewer scrutiny.

Reply:
Updated.